portal-impl/src/com/liferay/portlet/internal/PortletRequestImpl.java

```diff
@@ -718,29 +756,49 @@ else if (themeDisplay.isLifecycleAction()) {
 				!LiferayWindowState.isExclusive(request) &&
 				!LiferayWindowState.isPopUp(request)) {
 
-				if ((renderParameters == null) || renderParameters.isEmpty()) {
+				if ((privateRenderParameters == null) ||
+					privateRenderParameters.isEmpty()) {
+
 					RenderParametersPool.clear(request, plid, _portletName);
 				}
 				else {
 					RenderParametersPool.put(
-						request, plid, _portletName, renderParameters);
+						request, plid, _portletName, privateRenderParameters);
 				}
 			}
 		}
 		else {
-			Map<String, String[]> renderParameters = RenderParametersPool.get(
-				request, plid, _portletName);
+			Map<String, String[]> privateRenderParameters =
```

Comment:
@shuyangzhou I renamed the `renderParameters` variable to `privateRenderParameters` in order to remain consistent with Portlet 3.0's usage of the word "private" to distinguish between private and public render parameters.